ui/raidboss/data/05-shb/trial/hades-ex.js

```diff
@@ -25,7 +25,7 @@
   triggers: [
     {
       id: 'HadesEx Shadow Spread',
-      regex: / 14:47A8:Hades starts using Shadow Spread/,
+      regex: / 14:47A8:Hades starts using Shadow Spread on Hades\./,
```

Comment:
I still think it's better not to have these, both here and on the effects.  They're going to go away anyway if we switch to functions.  If you want to start adding example log lines, we can do that too.